RELEASE-NOTES.txt

```diff
@@ -15,6 +15,7 @@
 
 - Updated dependencies [#1759](https://github.com/Automattic/simplenote-electron/pull/1759)
 - Applied prettier formatting to all files [#1780](https://github.com/Automattic/simplenote-electron/pull/1780)
+- Transition TransitionDelayEnter to React hooks [#1784](https://github.com/Automattic/simplenote-electron/pull/1784)
```

Comment:
I would recommend a verb other than "Transition" as "Transition TransitionDelayEnter" is needlessly confusing. Maybe "Migrate"? But, does this need a release note at all?